docker/env/hydra.sh

```diff
@@ -95,4 +95,4 @@ docker run --rm ${TTY_STDIN} --privileged \
     --net=host \
     --name=${SCT_TEST_ID} \
     ${DOCKER_REPO}:${VERSION} \
-    /bin/bash -c "${TERM_SET_SIZE} eval '${CMD}'"
+    /bin/bash -c "sudo ln -s '${SCT_DIR}' '${WORK_DIR}'; ${TERM_SET_SIZE} eval '${CMD}'"
```

Comment:
It's unclear for me why you changed working directory.

Reply:
Problem is following - in order to run monitoring stack we need to start docker container from the inside of other docker container, problem is that we need to provide correct mounting paths, while we are inside of the container.
It could be done in two ways:
1. Having path mapping somewhere and translate docker pathes to host pathes eahc time you provide them to the docker. But you not always know or can separete path that is used localy or provided to the docker - example: you ran ./start-all.sh, and provide configuration file to it, it check if config file exists and run docker container with mapping this file to /etc/prometheuse/my_config.cfg. So, apperantly it will never work with this approach.
2. When hydra is running you map whole folder where sct is stored with exact same path. In this case while you working inseide of the directory you don't need to worry about docker volume mapping issue, since paths on the docker are the same as on the host. That is exactly what I did.

Only thing that, since we highly dependend on /sct folder we need to have a link to it.